.pipelines/azure-pipeline-build.yml

```diff
@@ -2,13 +2,15 @@ trigger:
   branches:
     include:
     - main
+    - netobsmetrics
```

Comment:
please remove this branch from this file

Reply:
removed